src/case-swapping.js

```diff
@@ -15,7 +15,10 @@
  * @return {string}
  */
 function swapCase(phrase) {
-  // write code here
+  return phrase.split('')
+    .map(symbol => symbol === symbol.toUpperCase()
+      ? symbol.toLowerCase() : symbol.toUpperCase())
```

Comment:
```suggestion
      ? symbol.toLowerCase() 
      : symbol.toUpperCase())
```
Перенеси варианты на новые строки